solidity/ecdsa/contracts/WalletRegistryGovernance.sol

```diff
@@ -1048,12 +1062,50 @@ contract WalletRegistryGovernance is Ownable {
             walletRegistry.dkgParameters().seedTimeout,
             walletRegistry.dkgParameters().resultChallengePeriodLength,
             walletRegistry.dkgParameters().resultSubmissionTimeout,
-            newSubmitterPrecedencePeriodLength
+            newSubmitterPrecedencePeriodLength,
+            walletRegistry.dkgParameters().resultChallengeExtraGas
         );
         dkgSubmitterPrecedencePeriodLengthChangeInitiated = 0;
         newSubmitterPrecedencePeriodLength = 0;
     }
 
+    /// @notice Begins the DKG result challenge extra gas update process.    .
+    /// @dev Can be called only by the contract owner.
+    /// @param _newDkgResultChallengeExtraGas New DKG result challenge extra gas
```

Comment:
The parameter description should end with a dot.

Reply:
Yeah. We were really inconsistent about it in `WalletRegistry` 😬 

Fixed in f337b2f7ae864a91261bb8fdcc957d22a2b12f58.